flake8_quotes/__init__.py

```diff
@@ -54,21 +41,29 @@ def _register_opt(parser, *args, **kwargs):
     @classmethod
     def add_options(cls, parser):
         cls._register_opt(parser, '--quotes', action='store',
-                          parse_from_config=True,
+                          parse_from_config=True, choices=cls.INLINE_QUOTES,
                           help='Deprecated alias for `--inline-quotes`')
         cls._register_opt(parser, '--inline-quotes', default='\'',
                           action='store', parse_from_config=True,
+                          choices=cls.INLINE_QUOTES,
                           help='Quote to expect in all files (default: \')')
+        cls._register_opt(parser, '--multiline-quotes', action='store',
+                          choices=cls.INLINE_QUOTES,
+                          help='Quote to expect in all files (default same as inline quote)')
 
     @classmethod
     def parse_options(cls, options):
         if hasattr(options, 'quotes') and options.quotes is not None:
             # https://docs.python.org/2/library/warnings.html#warnings.warn
             warnings.warn('flake8-quotes has deprecated `quotes` in favor of `inline-quotes`. '
                           'Please update your configugration')
-            cls.inline_quotes = cls.INLINE_QUOTES[options.quotes]
-        else:
-            cls.inline_quotes = cls.INLINE_QUOTES[options.inline_quotes]
+            options.inline_quotes = options.quotes
+
+        cls.inline_quotes = options.inline_quotes
+        if options.multiline_quotes is None:
+            options.multiline_quotes = options.inline_quotes
```

Comment:
We discussed this in #26, we want to support backwards compatibility so we won't lint multiline strings unless there is an explicit option to lint them.